tests/fsharp/core/printf/test.fsx

```diff
@@ -384,6 +384,71 @@ module CheckDisplayAttributes8 =
        override x.ToString() = "2"
 
     test "cenwoiwe8" (lazy(sprintf "%A" (Foo()))) "[1; 2]"
+
+// Check one returning two strings
+module CheckDisplayAttributes9 =
```

Comment:
instead of

```fsharp
// Check one returning two strings
module CheckDisplayAttributesUselessRandomNumber
```

you can just do

```fsharp
module ``Check DisplayAttributes one returning two strings`` =
```